scripts/upgrade_pysqlite.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Upgrade the version of pysqlite/sqlite to avoid crashes during testing.
+# Ideally, this code would just install a pre-built wheel - change it to this
+# once edX has its own artifact server.
```

Comment:
Is there a ticket for this followup work yet?

Reply:
I chose not to write one unless it looked like this PR would merge. I'll write one before hitting the merge button.